packages/common/test/http.test.js

```diff
@@ -1,281 +0,0 @@
-import nock from 'nock';
```

Comment:
Are we still using `nock`? We should have moved entirely to the `undici` pattern by now

Reply:
Removed